### ISSUE TITLE
Pull Request 837 (CSS Injection Fix) has not been pushed to npm repository

### ISSUE THREAD

**Issue by VidFerris** [jiahaog/nativefier#853]
### Description

The installation instructions tell the user to install via npm, but the current version of nativefier on npm doesn't have the patch to permit CSS injection with the new version of electron. Possibly worth providing alternative installation instructions to grab the version direct from this repository for when hotfixes don't get pushed, or adding another maintainer to the npm repo for when jiahaog is unavailable.

### Steps to reproduce issue

`npm install nativefier -g`

**Comment by zhouk** [comment]
Due to the CSS injection hotfix not being pushed, we're stuck with Electron 3 and 4, which has significant performance lags comparing more recent versions. 

I tried to clone the repo and build locally, which requires to use an older version of npm. I was able to successfully build it locally, but haven't figure out how to execute the local build to create apps yet. 

It will be super helpful if someone could provide some lean instructions.
